app/views/registration/steps/_details.html.haml

```diff
@@ -48,7 +48,7 @@
             .small-12.medium-4.large-4.columns
               .field
                 %label{ for: 'enterprise_state' } {{'registration_detail_state' | t}}
-                %select.chunky{ id: 'enterprise_state', name: 'state', ng: { model: 'enterprise.address.state_id', options: 's.id as s.abbr for s in enterprise.country.states', show: 'countryHasStates()', required: 'countryHasStates()' } }
+                %select.chunky{ id: 'enterprise_state', name: 'state', ng: { model: 'enterprise.address.state_id', options: "#{state_options_query}", show: 'countryHasStates()', required: 'countryHasStates()' } }
```

Comment:
Why not just `options: state_options_query`?

Reply:
Done. Thanks!